chat-components/ui/src/main/java/com/pubnub/components/chat/ui/component/message/renderer/GroupMessageRenderer.kt

```diff
@@ -78,32 +80,31 @@ object GroupMessageRenderer : MessageRenderer {
 
     @Composable
     override fun Separator(text: String) {
-        val separatorTheme = LocalMessageListTheme.current.separator
-        val messageTheme = LocalMessageListTheme.current.message.text
-        CompositionLocalProvider(
-            LocalContentAlpha provides ContentAlpha.disabled,
-        ) {
-            Row(
-                modifier = separatorTheme.modifier,
-                horizontalArrangement = Arrangement.Center,
-                verticalAlignment = Alignment.CenterVertically,
-            ) {
-                Divider(modifier = Modifier.weight(1f), color = messageTheme.color)
-                Spacer(modifier = Modifier.width(8.dp))
-
-                Text(text = text, theme = messageTheme)
-
-                Spacer(modifier = Modifier.width(8.dp))
-                Divider(modifier = Modifier.weight(1f), color = messageTheme.color)
-            }
-        }
+//        val separatorTheme = LocalMessageListTheme.current.separator
```

Comment:
Remove?

Reply:
This one should be restored in next components release (because it doesn't exists on other platforms yet)